infrastructure/helm/quantumserverless/charts/gateway/templates/deployment.yaml

```diff
@@ -42,14 +49,17 @@ spec:
             - name: http
               containerPort: {{ .Values.service.port }}
               protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /liveness
-              port: http
-          readinessProbe:
-            httpGet:
-              path: /readiness
-              port: http
+#          livenessProbe:
+#            httpGet:
+#              path: /liveness
+#              port: http
+#          readinessProbe:
+#            httpGet:
+#              path: /readiness
+#              port: http
+          volumeMounts:
```

Comment:
Are there issues with probes?

Reply:
yeah, I had issues with probes on local rancher setup. It hangs with unready status